<!DOCTYPE html>
<html lang="en">
  <head>
	<meta charset="utf-8">
	<meta name="viewport"    content="width=device-width, initial-scale=1.0">
	<meta name="description" content="">
	<meta name="author"      content="Raymond Cheng">
    
	<title>Home | Raymond Cheng </title>

  <!-- showing HCC icon thumbnail -->
	<link rel="shortcut icon" href="image/HCC_icon.png">
	
    <!-- Bootstrap stylesheet (pre-written style of webpage)-->
	<link href="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.no-icons.min.css" rel="stylesheet">
	
    <!-- Icons for social media-->
    <!-- Font Awesome icons on your webpage, allowing you to easily use icons like fa-twitter, fa-github -->
    <!-- Use defer to ensure the script won't delay rendering the webpage, which improves user experience, especially for content-heavy pages -->
    <script defer src="https://use.fontawesome.com/releases/v6.7.0/js/all.js" crossorigin="anonymous"></script>
    <!-- Icons for academic links-->
    <link href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css" rel="stylesheet">
      
    <!-- Fonts -->
	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Alice|Open+Sans:400,300,700">
	
    <!-- Custom styles -->
    <link rel="stylesheet" href="css/styles.min_modified.css">

  </head>
  
  <body>

    
 <header id="header">
    <style>
  #head {
    background-image: linear-gradient(rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.5)), 
                      url('image/phylogenetic_comic.png');
    background-size: cover;
    background-position: center;
    background-repeat: no-repeat;
    height: 400px; /* Adjust height as needed */
  }
</style>

<div id="head" class="parallax" data-parallax-speed="10">
      <h1 id="logo" class="text-center">
      <img class='img-circle' src="image/Raymond_headphoto.png" alt="">
      <span class="title">Hsueh-Chien Raymond Cheng</span>
      <span class="tagline">  <br>
				<a href="mailto:#">raymond.cheng [at] sanger.ac.uk </a></span>
      <span class="follow">
        <center>
            <a href="https://github.com/hsuehchien66"><i class="fab fa-github fa-1x"></i></a> 
            <a href="https://scholar.google.com/citations?user=brJlMVQAAAAJ&hl=en"> <i class="ai ai-google-scholar ai-2"></i></a>
	    <a href="https://www.linkedin.com/in/hsueh-chien-raymond-cheng-9b1191249/"><i class="fab fa-linkedin fa-1x"></i></a>
        </center>
      </span>
      <span class="tagline">
          <center>
              <a href="index.html" style="font-size:1rem"> English</a>
              <a style="font-size:1.1rem"> | </a>
              <!-- Add Traditional Chinese Page -->
              <!-- a href="index_french.html" style="font-size:1rem"> Traditional Chinese </a> -->
          </center>   
      </span>    
   </h1>
</div>

<nav class="navbar navbar-default navbar-sticky">
    <div class="container-fluid">

        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="true">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
        </div>

        <div class="navbar-collapse collapse" id="bs-example-navbar-collapse-1">

            <ul class="nav navbar-nav">
            
                
                <li>
                    <a href="index.html">Home</a>
                </li>
                
            
                
                <li class="active">
                    <a href="cv.html">CV</a>
                </li>
                
            
                
                <li>
                    <a href="publications.html">Publications</a>
                </li>
                
            
            </ul>

        </div> 
        
    </div>
</nav>

</header>
    
 
    
  <main id="main">
  
  	<div class="container">
  
  		<div class="row topspace">
  
  			
  			<aside class="col-sm-3 sidebar sidebar-right">
                <br>
  				<div class="nav text-right nav-side">
  					<li style="color: purple !important;" class="active"><a href="#education" style="color: purple !important;">Education</a></li>
                    			<li style="color: purple !important;"><a href="#experience" style="color: purple !important;">Research Experience</a></li>
					<li style="color: purple !important;"><a href="#award" style="color: purple !important;">Awards and Honor</a></li>
  				</div>
  
  			</aside>
  			
  
  			
  			<article class="col-sm-9 maincontent">
               
                
                <h3 id="education">Education</h3>
                 

                <p  align="justify"> <b> 2021-2025: PhD in Biology, Wellcome Sanger Institute, University of Cambridge, UK. </b> <br> Supervised by Prof. Stephen Bentley, Dr. John Lees and Prof. Henrik Salje <br> <i> Quantifying the invasiveness and spread of pneumococcus using genomic data. </i> <br> My main PhD work was building mathematical models and utilising whole genome sequenceing data to understand the disease progression and  transmission dynamics of pneumococcal diseases. I developed methods that make use of the growing availability of both genomic and spatial data.  </p>

                <p  align="justify"> <b> 2018-2019: Master’s degree – Genome Sciences, National Yang-Ming University, Taipei, Taiwan </b> <br> Supervised by Dr. Wen-Ya Ko <br> <i> Identifying genomic signatures of natural selection and the implications in disease susceptibility of the Taiwanese people </i> <br> My main Master work was investigating the signatures of natural selection in human genomes using statistical tests to explore the rates of substitutions and unique haplotype structures. </p>
                
                
                <h3 id="experience">Research experience</h3>
                
                <p  align="justify"> <b> 2021-Now: PhD student, Wellcome Sanger Institute, University of Cambridge, UK </b> <br>  Funded by Wellcome Sanger Institute 4-year PhD Programme. Supervised by Prof. Stephen Bentley.  </p>
                
                <p  align="justify"> <b> 2020-2021: Research assistant, Academia Sinica, Taiwan. </b> <br>  Working with Prof. Chung-Yen Lin.
				<ul> <li> Establish algorithm of <i> de novo </i> Human genome assembly utilising both 10X genomic and Illumina data </li> 
				     <li> Designing SNP array platform for aquatic species in Taiwan on the purpose of selective and sustainable breeding </li>
				     <li> Constructing deep neural networks to improve whole genome variant calling and prediction on obesity </li>
				</ul>  </p>
                
                <p  align="justify"> <b> 2017-2018: Research intern, National Yang-Ming University, Taipei, Taiwan </b> <br> Supervised by Prof. Wen-Ya Ko <br> <i> Molecular evolution of the S-crystallin gene family in squids </i> </p>
                
                <h3 id="award">Awards & Honor</h3>
		<p  align="justify"> 
			2024 Invited talk, National Health Research Institute, Taiwan
			<ul><li> 2024 Conference talk, International Society of Pneumonia and Pneumococcal Diseases (ISPPD-13) conference </li>
			<li> 2023 Travel Award, Asian Pneumococcal Symposium (APS-4) </li>
			<li> 2022 Taiwan Ministry of Education schloarship </li>
			<li> 2021 Wellcome Sanger Institute PhD scholarship </li>
			<li> 2018 Taiwan BioGenome program scholarship, Academia Sinica grant 23-23 </li>
			<li> 2016 Academia Achievement Award, National Yang Ming University </li>
			<li> 2015 Golden Medal, iGEM competition </li></ul></p> 

		<h3 id="intro">Brief Introduction</h3>
		<p  align="justify"> 
			Raymond is a PhD student at the Wellcome Sanger Institute in Prof. Stephen Bentley’s
group and working closely with Prof. Henrik Salje in the University of Cambridge and Dr.
John Lees at the European Bioinformatic Institute. He is focusing on quantifying the
invasiveness of Streptococcus pneumoniae with different genetic backgrounds and
host environments by using the whole genome sequence data among multiple
populations. He is also interested in studying various modes of evolution including
migration and selection in pathogenic bacteria and how they shape different population
structures. </p>

  			</article>
  
  		</div>
  	</div>	
  
  </main>

    
    
<footer id="footer">
	<div class="container">
		<div class="row">
            <div class="col-md-12 widget">
                <center>
                <h3 class="widget-title">Contact</h3>
                <a href="mailto:#">raymond.cheng [at] sanger.ac.uk </a><br>
				<br>
				Department of Parasites and Microbes <br> Wellcome Sanger Institute <br> University of Cambridge <br> United Kingdom
                <br>
                </center>
            </div>
		</div> 
	</div>
</footer> 

<footer id="footer">
	<div class="container">
		<div class="row">
			<div class="col-md-12 widget">
				<div class="widget-body">
					<p class="text-center">
						Copyright &copy; 2024, Raymond Cheng<br> 
                        Last update: December 2024<br> 
						Design: <a href="http://www.gettemplate.com" rel="designer">Initio by GetTemplate</a> </p>
				</div>
			</div>
		</div> <!-- /row of widgets -->
	</div>
</footer>



<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
<script src="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>




<script src="https://miguelsimoni.github.io/hugo-initio-site/js/bundle.min.f4a965ad0a8118e32f8f0f158ff3aadbacf700934c22286a1a5b245105e9006da73a873b001a160db22498909c1df14d1f835dba5ad76f80b32b0234182b2a58.js" integrity="sha512-9KllrQqBGOMvjw8Vj/Oq26z3AJNMIihqGlskUQXpAG2nOoc7ABoWDbIkmJCcHfFNH4NdulrXb4CzKwI0GCsqWA=="></script>

<script id="dsq-count-scr" src="//hugo-initio-site.disqus.com/count.js" async></script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'GA-000000000-0', 'auto');
  ga('send', 'pageview');
</script>

</body>
</html>

    
  </body>
  
</html>
